static/app/views/detectors/components/detailsPanel.tsx

```diff
@@ -0,0 +1,50 @@
+import styled from '@emotion/styled';
+
+import {Flex} from 'sentry/components/container/flex';
+import {Container} from 'sentry/components/workflowEngine/ui/container';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+
+// TODO: Make component flexible for different alert types
+function DetailsPanel() {
+  return (
+    <Container>
+      <Flex column gap={space(0.5)}>
+        <Heading>{t('Query:')}</Heading>
+        <Query>
+          <Label>{t('visualize:')}</Label> <Value>{t('p75')}</Value>
+          <Label>{t('where:')}</Label> <Value>{t('device.name is "Chrome"')}</Value>
+          <Label>{t('grouped by:')}</Label> <Value>{t('release')}</Value>
+        </Query>
```

Comment:
Consider using the [`dl`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl), [`dt`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dt), and [`dd`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dd) elements for improved semantics here